### PR TITLE
Document how to locally run Athenian

### PR DESCRIPTION
**Pull request by dpordomingo**
They're needed 2 different Apps:
- one for Athenian API,
- one for the SPA.

### CODE REVIEW

**Comment by dpordomingo** [comment]
I locally tested this with a brand new Auth0 account, and it worked, but since you @dennwc already spent a lot of time with Auth0, you could maybe confirm that I didn't miss any other major thing for development purposes.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -37,22 +37,41 @@ The following commands auto generate different parts of the app:
 
 For locally testing the authentication, you need to configure Auth0 and GitHub to work together. Follow these steps or refer to Auth0 and GitHub docs.
 
-#### 1. Register Single-Page Applications in Auth0 ([docs](https://auth0.com/docs/dashboard/guides/applications/register-app-spa))
+You also need to run the [Athenian API](https://github.com/athenianco/athenian-api) and configure it in order to connect with your "Auth0 Management API"
```
Comment (vmarkovtsev) [inline docs/CONTRIBUTING.md]:
```suggestion
You also need to run the [Athenian API](https://github.com/athenianco/athenian-api) and configure it in order to connect to your "Auth0 Management API"
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -37,22 +37,41 @@ The following commands auto generate different parts of the app:
...
     - "Homepage URL": `http://localhost:3000`
     - "Authorization callback URL": `https://<auth0 domain>/login/callback` (e.g., `your-app.eu.auth0.com`)
 2. Enable the connection with GitHub from your Auth0 Dashboard providing the `Client ID` and `Client Secret` of the GitHub OAuth App from the previous step, and ask for these permissions:
     - `read:user`
 
-#### 3. Configure Athenian App
+#### 4. Configure Athenian WebApp
+
+Update the [`config.js`](https://github.com/athenianco/athenian-webapp/blob/master/public/config.js) file:
+- `auth.domain`, `auth.clientId`, `auth.redirectUri` and `auth.logoutRedirectUri` with the corresponding values of your Auth0 Single Page Application created by `1`.
```
[the remark this answers is not in the document]
Comment (dpordomingo) [reply docs/CONTRIBUTING.md]:
```suggestion
- `auth.domain`, `auth.clientId`, `auth.redirectUri` and `auth.logoutRedirectUri` with the corresponding values of your Auth0 Single Page Application created by [#1](#1-create-a-single-page-applications-in-auth0-docs).
```

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -37,22 +37,41 @@ The following commands auto generate different parts of the app:
...
+- `auth.audience` will be the one for your "Auth0 Management API" (e.g. https://your-tenant.eu.auth0.com/api/v2/).
+- `api.basePath` must be the url of your [Athenian API](https://github.com/athenianco/athenian-api)
+
 
-Update the [`config.js`](https://github.com/athenianco/athenian-webapp/blob/master/public/config.js) file by setting both `auth.domain`, `auth.clientId`, `auth.redirectUri` and `auth.logoutRedirectUri` with the corresponding values of the Auth0 application.
+#### 5. Configure Athenian API
+Configure your [Athenian API](https://github.com/athenianco/athenian-api) (using the `.env` file if you're runing it locally):
+- `AUTH0_DOMAIN`, `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET` with the corresponding values of your Auth0 Machine to Machin Application created by `2`.
```
[the remark this answers is not in the document]
Comment (dpordomingo) [reply docs/CONTRIBUTING.md]:
```suggestion
- `AUTH0_DOMAIN`, `AUTH0_CLIENT_ID` and `AUTH0_CLIENT_SECRET` with the corresponding values of your Auth0 Machine to Machine Application created in [#2](#2-create-a-machine-to-machine-applications-in-auth0-docs).
```

**Comment by dpordomingo** [comment]
@vmarkovtsev your suggestions were addressed by 2f6cc2b. Many thanks for that in-depth review!!

**Pull request merged**